index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Hello-World</title>
+	</head>
+	<body>
+		<h1 class="header">"Hello Microverse!"</h1>
+	</body>
```

Comment:
- _(optional)_ Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO